### DISCUSSION TITLE
Should we have a mental wellbeing section?

### DISCUSSION THREAD

**Discussion by OscartGiles** [alan-turing-institute/REG-handbook#6] [Ideas]
Dave Horsfall (SSI fellow, Newcastle University) gave a great talk to REG on mental wellbeing in the workplace. This sparked the idea that our REG handbook should have a section on mental wellbeing. 

Few starter points for discussion:
- How REG can help if you're feeling overwhelmed or suffering high levels of stress.
- Provide information on workload management
     - People sometimes need a strong reminder (from new starters to seasoned leaders) that the answer to "I've got so much to do, I can't possibly do it all in my working hours", should not be "I'll just work every waking hour" rather than getting help. 
- How you can help other people in the team to maintain a healthy working culture that promotes mental wellbeing. 
- Sign-post the services you can access through Turing (external supervision, BUPA mental health etc)

**Comment by JimMadge** [comment]
Hi @OscartGiles,

Great idea. I think this should be part of the handbook.

Here are some thoughts.

> - How REG can help if you're feeling overwhelmed or suffering high levels of stress.

Maybe the leadership team would have some input here. Is there anything on the wiki?

> - Provide information on workload management 
> - How you can help other people in the team to maintain a healthy working culture that promotes mental wellbeing.

We should ask Dave Horsfall if he has any good recommendations or resources we can use or refer to.

> - Sign-post the services you can access through Turing (external supervision, BUPA mental health etc)

Are there stable links for these? If so those should go into the book

Would you like to lead on this?
